tmt/steps/execute/__init__.py

```diff
@@ -211,33 +223,37 @@ def base_command(
         @click.option(
             '-h', '--how', metavar='METHOD',
             help='Use specified method for test execution.')
-        def execute(context, **kwargs):
+        def execute(context: Any, **kwargs: Any) -> None:
             context.obj.steps.add('execute')
             Execute._save_context(context)
 
         return execute
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
         # Add option to exit after the first test failure
-        options = [click.option(
+        return cast(List[tmt.options.ClickOptionDecoratorType], [click.option(
```

Comment:
We should try to get rid of this cast(). PR https://github.com/teemtee/tmt/pull/1324 has https://github.com/teemtee/tmt/pull/1324#discussion_r926567419 on how to do that, so I'll wait if it gets approved before changing it here also.